### PR TITLE
[Magic Firewall] Replacing API directions with UI

### PR DESCRIPTION
**Pull request by dcpena**
Replacing API guidance with instructions for completing the same tasks via the dash.

Addresses [PCX-1575](https://jira.cfops.it/browse/PCX-1575)

### CODE REVIEW

File: products/magic-transit/src/content/magic-firewall/index.md [hunk 1]
```diff
@@ -6,16 +6,13 @@ order: 1
 # Magic Firewall
 
 Magic Firewall is a network-level firewall delivered through Cloudflare to protect your network. It is
-configurable by all Magic Transit customers by making calls to Cloudflare's API. Rules can be crafted in
-Wireshark syntax, a domain specific language common in the networking world and the same syntax we use
-across our other products. With this syntax, you can easily craft extremely powerful rules to precisely
-allow or deny any traffic in or out of your network.
+configurable by all Magic Transit customers from the dashboard under **Firewall Rulesets** or by making calls to Cloudflare's API. Rules can be crafted in Wireshark syntax, a domain specific language common in the networking world and the same syntax we use across our other products. With this syntax, you can easily craft extremely powerful rules to precisely allow or deny any traffic in or out of your network.
```
Comment (crwaters16) [inline products/magic-transit/src/content/magic-firewall/index.md]: -hyphenate domain-specific
-extremely might be superfluous, though I appreciate the emphasis :)

File: products/magic-transit/src/content/magic-firewall/rules.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Rules
+order:  2
+---
+
+# Rules
+
+You can check for an existing root ruleset from the dashboard or via the [Account rulesets API](https://api.cloudflare.com/#account-rulesets-properties). If you're a new Magic Transit customer, you may not have a root ruleset created for your account.
```
Comment (crwaters16) [inline products/magic-transit/src/content/magic-firewall/rules.md]: contraction

File: products/magic-transit/src/content/magic-firewall/rules.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+1. Fill out the information for your new rule.
+1. When you are done, click **Add new rule**.
+
+## Update rules
+
+To update a rule:
+
+1. From your dashboard, click **Firewall Rulesets**.
+1. Click **Magic Firewall**. A list of firewall rules displays.
+1. Locate the rule you'd like to edit and click **Edit**.
```
Comment (crwaters16) [inline products/magic-transit/src/content/magic-firewall/rules.md]: contraction - consider "you want to edit" for easy way to remove.

**Review by crwaters16 — APPROVED** [review]

File: products/magic-transit/src/content/magic-firewall/index.md (hunk 1 above)
Comment (dcpena) [reply products/magic-transit/src/content/magic-firewall/index.md]: I did not realize "extremely" was in there, haha. Removed!

File: products/magic-transit/src/content/magic-firewall/rules.md (hunk 2 above)
Comment (dcpena) [reply products/magic-transit/src/content/magic-firewall/rules.md]: Fixed

File: products/magic-transit/src/content/magic-firewall/rules.md (hunk 3 above)
Comment (dcpena) [reply products/magic-transit/src/content/magic-firewall/rules.md]: Much better! thanks :)

**Pull request merged**